### DISCUSSION TITLE
Can someone help me, my profile repository is retired what does it mean?

### DISCUSSION THREAD

**Discussion by gospacedev** [community/community#12799] [Projects and Issues] [answered]
Can someone help me, my profile repository is retired what does it mean?

**Comment by airtower-luna** [comment, marked as answer]
Did you recently change your username? The GitHub documentation describes how repository names become retired in the article about username changes:
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username
